chore-no-more/src/components/Groups.js

```diff
@@ -1,99 +1,108 @@
 import { useState, useEffect } from 'react'
-import { supabase } from '../supabaseClient'
-import { useSession } from '@supabase/auth-helpers-react';
+import { supabase } from '../supabaseClient';
+import { useSession} from '@supabase/auth-helpers-react';
 
 
 export function Groups() {
-    const [ groupsNames, setGroupNames] = useState([]);
-    const [ groupName, setGroupName] = useState('');
-    const session = useSession();
+  const [groupsName, setGroupName] = useState();
+  const [Group, setGroup] = useState('');
+  const session = useSession();
+  const [exist, setExist] = useState(false); // Initialize to false instead of null
+  const [loading, setLoading] = useState(true); // Add loading state
 
-    async function addGroup(count){
-      const { error } = await supabase
-      .rpc('append_array', {new_element: count, email: session.user.email});
-      if(error) {
-        alert("Adding group ID to user table is not working");
-        console.log(error);
+  useEffect(() => {
+    async function getGroupId() {
+      console.log(session.user.email);
```

Comment:
I would probably take out these logs if we don't need them anymore, even if just for the main branch, just so the console stays clean. Keep the error console logs for sure, but we probably don't want random data to print at every run